packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -12,6 +12,15 @@ const queue = require(`async/queue`)
 const path = require(`path`)
 const existsSync = require(`fs-exists-cached`).sync
 
+const digestObject = obj => crypto
```

Comment:
This is actually available to plugins as `createContentDigest` so I'd suggest passing it similarly to how we're passing cache!